src/test/scala/edu/ie3/simona/agent/participant/EvcsAgentModelCalculationSpec.scala

```diff
@@ -1609,7 +1609,9 @@ class EvcsAgentModelCalculationSpec
             ) =>
           modelUuid shouldBe evcsInputModelQv.getUuid
           refPower shouldBe combinedChargingPowerSq
-          minPower shouldBe ev4500.sRatedAc * -1 // battery of earlier ev is above lowest soc now
+          minPower shouldBe ev4500.sRatedAc.toPower(
+            1.0
+          ) * -1 // battery of earlier ev is above lowest soc now
```

Comment:
In general, if you place comments _above_ and not _after_ the code that they reference, there's less of a chance that spotless breaks up the code in this ugly way. Please check all changes again for this.

```suggestion
          // battery of earlier ev is above lowest soc now
          minPower shouldBe ev4500.sRatedAc.toPower(1.0) * -1
```